functions/utils/getSetup.ts

```diff
@@ -0,0 +1,15 @@
+const watermark = 'https://app.uniswap.org/images/324x74_App_Watermark.png'
+const check = 'https://app.uniswap.org/images/54x54_Verified_Check.svg'
```

Comment:
These should be exported as constants, maybe from `functions/constants.ts`?
Also, uppercase the constants: WATERMARK_URL, CHECK_URL.